frontends/web/src/utils/transport-qt.ts

```diff
@@ -51,7 +51,7 @@ async function initTransport() {
   return webChannel;
 }
 
-export function call(query: string) {
+export function call(query: string): Promise<any> {
```

Comment:
Also here, removing it shows me `Promise<unknown>` which should be stricter.

![Screen Shot 2023-12-24 at 10 27 32](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/5dcc4a5c-f786-46df-b561-d841981837b1)

Reply:
Done, but strange that `apiGet()` and `apiPost()`, which return `call(...)`, have `Promise<any>` and it does not seem to work if changed `Promise<unknown>`.

Reply:
true. I think we could return `Promise<unknown>` for apiGet and apiPost, but then we need to rewrite all api modules and type the return `as Promise<Type>`

```diff
diff --git a/frontends/web/src/api/account.ts b/frontends/web/src/api/account.ts
index 6323214db..1029e95b2 100644
--- a/frontends/web/src/api/account.ts
+++ b/frontends/web/src/api/account.ts
@@ -80,8 +80,8 @@ type TEthAccountCodeAndNameByAddress = SuccessResponse & {
   name: string;
 }
 
-export const getEthAccountCodeAndNameByAddress = (address: string): Promise<TEthAccountCodeAndNameByAddress> => {
-  return apiPost('accounts/eth-account-code', { address });
+export const getEthAccountCodeAndNameByAddress = (address: string) => {
+  return apiPost('accounts/eth-account-code', { address }) as Promise<TEthAccountCodeAndNameByAddress>;
 };
 
 export interface IStatus {
@@ -91,8 +91,8 @@ export interface IStatus {
     offlineError: string | null;
 }
 
-export const getStatus = (code: AccountCode): Promise<IStatus> => {
-  return apiGet(`account/${code}/status`);
+export const getStatus = (code: AccountCode) => {
+  return apiGet(`account/${code}/status`) as Promise<IStatus>;
 };
 ```

But I am not sure if this improves something.

Reply:
that would look like a step backwards actually